docs/core/custom-models.md

```diff
@@ -100,8 +100,56 @@ of this you need to specify a so called 'custom model', which is a set of rules
 `priority` of an edge. The custom model is written in JSON language and also includes a few more parameters like the
 `distance_influence`.
 
-Setting up rules for `speed` and `priority` is very similar, so in our examples we will first concentrate on the `speed`
-rules, but as you will see they can be applied very much the same way for priority as well.
+Here is a complete request example for a POST /route query in berlin that includes a custom model:
+
+```json
+{
+  "points": [
+    [
+      13.31543,
+      52.509535
+    ],
+    [
+      13.29779,
+      52.512434
+    ]
+  ],
+  "profile": "car",
+  "custom_model": {
+    "speed": [
+      {
+        "if": true,
+        "limit_to": 100
+      }
+    ],
+    "priority": [
+      {
+        "if": "road_class == MOTORWAY",
+        "multiply_by": 0
+      }
+    ],
+    "distance_influence": 50
+  }
+} 
+```
+
+If you put this JSON into a file `custom.json` you can send the request like this:
+
+```shell
+curl -XPOST -H "Content-Type: application/json" "http://localhost:8989/route" --data custom.json
```

Comment:
Just for later reference - there was a mistake and it should be `--data @custom.json`.